e1000d/src/device.rs

```diff
@@ -6,6 +7,9 @@ use syscall::flag::O_NONBLOCK;
 use syscall::io::Dma;
 use syscall::scheme::Scheme;
 
+// TODO: One // line here to describe what this GROUP of constants is
+// TODO: one /// line describing each constant. Just defining the acronym
```

Comment:
if we could just give the link to that at the top of the document and then summarize what each group is doing that would probably be sufficient.